app/src/main/java/io/github/droidkaigi/confsched2018/presentation/common/binding/TextBinding.kt

```diff
@@ -45,8 +46,12 @@ fun TextView.setPeriodText(startDate: Date?, endDate: Date?) {
 @BindingAdapter(value = ["prefix", "roomName"])
 fun TextView.setRoomText(prefix: String?, roomName: String?) {
     prefix ?: return
-    text = when (roomName) { null -> ""
-        else -> context.getString(R.string.room_format, prefix, roomName)
+    roomName ?: return
+    val isLayoutDirectionRtl = resources.configuration.layoutDirection == View.LAYOUT_DIRECTION_RTL
```

Comment:
Could you extract this comparison as an extension method of View?

Reply:
done with https://github.com/DroidKaigi/conference-app-2018/pull/606/commits/59370504bff1d452dd5409bab9e6ccde425be948